tests/test_instrument_all_and_run.py

```diff
@@ -157,15 +159,22 @@ def test_sort():
                 )
             ]
         )
-        test_results, coverage_data = opt.run_and_parse_tests(
+        test_results, coverage_data = func_optimizer.run_and_parse_tests(
             testing_type=TestingMode.BEHAVIOR,
             test_env=test_env,
-            test_files=test_files,
+            test_files=func_optimizer.test_files,
             optimization_iteration=0,
             pytest_min_loops=1,
             pytest_max_loops=1,
             testing_time=0.1,
         )
+
+        out_str = """codeflash stdout: Sorting list
+result: [0, 1, 2, 3, 4, 5]
+
+codeflash stdout: Sorting list
+result: [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]"""
+        assert out_str == test_results[0].stdout.strip()
```

Comment:
we should not strip here. The stdout must be exactly what is printed. the whitespace matters